indico/web/forms/fields/simple.py

```diff
@@ -147,5 +147,13 @@ def _value(self):
         return self.text_value
 
 
+class IndicoEmailRecipientsField(IndicoStaticTextField):
+    count = None
+    widget = JinjaWidget('forms/email_recipients_widget.html', single_kwargs=True)
+
+    def _count(self):
```

Comment:
Besides this property having to be set from outside (wouldn't be necessary when you pass a list ;)), this method is slighly pointless. You can just get `field.count` in the template. I guess you wanted to remain consistent with `_value()` but that's already not the prettiest thing (not sure why WTForms does this) so let's not add more of this